test/t_second_doc.ref

```diff
@@ -1,4 +1,4 @@
->>>REQFLOW stat -s -c t_second_doc.req
+>>> ! REQFLOW stat -s -c t_second_doc.req
```

Comment:
not needed after removal of !

Reply:
Yes but I do not know a way to remove it.